docs/hydra/guides/digital-ocean.mdx

```diff
@@ -0,0 +1,465 @@
+bash /tmp/nodesource_setup.sh
+apt-get install nodejs npm jq unzip
+```
+
+Confirm the correct version of NodeJs is installed:
+
+```bash
+node -v
+v16.14.2
+```
+
+## Install PostgreSQL
+
+Install PostgreSQL by running the following command:
+
+```bash
+sudo apt install postgresql postgresql-contrib
+sudo -i -u postgres
+```
+
+Create the database:
+
+```bash
+createdb hydra
+```
+
+Change the default password encryption to a stronger one:
+
+```bash
+psql
+# Change the default password encryption to stronger one
+ALTER SYSTEM SET password_encryption = 'scram-sha-256';
+SELECT pg_reload_conf();
+```
+
+Create a temporary user to get a hash of a password:
+
+```bash
+CREATE USER tmp_user_to_create_a_password WITH PASSWORD 'b0qw68gr3Q';
+# Copy the password hash from the following command
+SELECT rolpassword FROM pg_catalog.pg_authid WHERE rolname='tmp_user_to_create_a_password';
+# Delete the temporary user
+DROP USER IF EXISTS tmp_user_to_create_a_password;
+```
+
+Now we can create a user by running the following command, fill in the password
+hash you copied in the step before:
+
+```bash
+CREATE USER hydra with password 'SCRAM-SHA-256$4096:<your-copied-hashed-password-here>';
+GRANT CONNECT ON DATABASE hydra to hydra;
+```
+
+Exit psql:
+
+```bash
+\q
+```
+
+We need to change the `/etc/postgresql/12/main/pg_hba.conf` file and append the
+following to enable `scram-sha-256` encryption:
+
+```bash
+host    all             all             127.0.0.1/32            scram-sha-256
+```
+
+Check your credentials against PostgreSQL:
+
+```bash
+psql -U hydra -W -h 127.0.0.1
+Password:
+psql (12.9 (Ubuntu 12.9-0ubuntu0.20.04.1))
+SSL connection (protocol: TLSv1.3, cipher: TLS_AES_256_GCM_SHA384, bits: 256, compression: off)
+Type "help" for help.
+```
+
+We installed and configured the PostgreSQL database. 
+Next we will set up Ory Hydra.
+
+## Install Ory Hydra
+
+For security reasons, we need to create a new user with prohibited login:
+
+```bash
+useradd -s /bin/false -m -d /opt/hydra hydra
+```
+
+Create a couple of folders:
+
+```bash
+mkdir /opt/hydra/{bin,config}
+```
+
+Install Ory Hydra:
+
+```bash
+cd /opt/hydra/bin
+# Download a new version of Ory Hydra
+wget https://github.com/ory/hydra/releases/download/${useLatestRelease('hydra')}/hydra_${useLatestReleaseFilename('hydra')}-linux_64bit.tar.gz
+# Extract contents
+tar xfvz hydra_${useLatestReleaseFilename('hydra')}-linux_64bit.tar.gz
+# Remove redundant files
+rm *md
+rm LICENSE`}
+```
+
+Download the [Quickstart](../5min-tutorial) configuration files:
+
+<CodeBlock className="language-shell">{`cd ../config
+wget https://raw.githubusercontent.com/ory/hydra/${useLatestRelease(
+  'hydra'
+)}/contrib/quickstart/5-min/hydra.yml`}</CodeBlock>
+
+We need to add some configuration in `hydra.yml` to use our Postgres
+
+```yml
+dsn: postgres://hydra:b0qw68gr3Q@127.0.0.1:5432/hydra?sslmode=disable&max_conns=20&max_idle_conns=4
+urls:
+  self:
+    issuer: https://accounts.gen1us2k.com
```

Comment:
what is this part for? would be good to explain it

the DSN setting we can also mention, to connect the database.